python/pylibraft/pylibraft/test/test_distance.py

```diff
@@ -57,7 +58,10 @@ def test_distance(n_rows, n_cols, inplace, metric, order, dtype):
 
     output = np.zeros((n_rows, n_rows), dtype=dtype)
 
-    expected = cdist(input1, input1, metric)
+    if metric == "inner_product":
```

Comment:
We mentioned offline, we should test this through both the C++ and python APIs [here](https://github.com/rapidsai/raft/tree/branch-23.04/cpp/test/distance). If you think it'd be helpful, I don't mind helping to add a quick google test case here.

Reply:
thanks for the pointers! I can't believe I missed the c++ tests for the other metrics (didn't see them when I first looked for some , though its super obvious they are there). Added in the last commit -

Reply:
Haha no problem. I just know a lot of things have been filling up people's plates lately so I didn't mind jumping in to help if needed.